packages/ketcher-react/src/script/ui/state/saltsAndSolvents/index.ts

```diff
@@ -0,0 +1,76 @@
+/****************************************************************************
+ * Copyright 2021 EPAM Systems
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ ***************************************************************************/
+
+import { AnyAction } from 'redux'
+import { appUpdate } from '../options'
+import {
+  SaltsAndSolventsProvider,
+  FunctionalGroupsProvider,
+  SdfItem,
+  SdfSerializer,
+  Struct
+} from 'ketcher-core'
+import { prefetchStatic } from '../templates/init-lib'
+
+interface SaltsAndSolventsState {
+  lib: []
+  mode: string
+}
+
+const initialState: SaltsAndSolventsState = {
+  lib: [],
+  mode: 'fg'
+}
+
+const saltsAndSolventsReducer = (
+  state = initialState,
+  { type, payload }: AnyAction
+) => {
+  switch (type) {
+    case 'SALTS_AND_SOLVENTS_INIT':
```

Comment:
would be good to eventually move these to constants

Reply:
Yes, I agree